thunder/tests/test_dynamo.py

```diff
@@ -551,7 +552,10 @@ def forward(self, x):
     out.backward(g)
 
     ref_g = torch.ones_like(ref_out)
-    ref_out.backward(ref_g)
+    with warnings.catch_warnings():
+        # FutureWarning: `torch.cpu.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cpu', args...)` instead.
+        warnings.filterwarnings("ignore", category=FutureWarning)
+        ref_out.backward(ref_g)
```

Comment:
can't we use [`@pytest.mark.filterwarnings`](https://docs.pytest.org/en/stable/how-to/capture-warnings.html#filterwarnings) instead?